src/components/input/inputStyles.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+
+
+export const inputStyles = StyleSheet.create({
```

Comment:
This file should have .ts extension. Because we don't use JSX syntax here.
Also we don't need import React here.